hack/generator/pkg/conversions/property_conversions.go

```diff
@@ -145,6 +145,15 @@ func CreateTypeConversion(
 	return nil, err
 }
 
+
+func NameOfPropertyAssignmentFunction(name astmodel.TypeName, direction Direction, idFactory astmodel.IdentifierFactory) string {
+	nameOfOtherType := idFactory.CreateIdentifier(name.Name(), astmodel.Exported)
+	return direction.SelectString(
+		"AssignPropertiesFrom"+nameOfOtherType,
+		"AssignPropertiesTo"+nameOfOtherType)
```

Comment:
Maybe clearer as:
```suggestion
	return "AssignProperties" + direction.SelectString("From", "To") + nameOfOtherType
```

Reply:
Actually maybe `.SelectString("From", "To")` could be built-in? Although that's not too bad as-is.

Reply:
I'm doing a Code Gardening PR that cleans up a few things like this, will include there.